source/extensions/filters/common/ratelimit/ratelimit_impl.cc

```diff
@@ -110,9 +110,10 @@ void GrpcClientImpl::onSuccess(
   callbacks_ = nullptr;
 }
 
-void GrpcClientImpl::onFailure(Grpc::Status::GrpcStatus status, const std::string&,
+void GrpcClientImpl::onFailure(Grpc::Status::GrpcStatus status, const std::string& msg,
                                Tracing::Span&) {
   ASSERT(status != Grpc::Status::WellKnownGrpcStatus::Ok);
+  ENVOY_LOG_TO_LOGGER(Logger::Registry::getLog(Logger::Id::filter), warn, "rate limit fail, status={} msg={}", status, msg)
```

Comment:
I haven't seen this pattern anywhere else in the code base, from a quick code search I see that most cases in `source/extensions` use the `ENVOY_LOG` macro

Reply:
How would that log look like?

Reply:
`ENVOY_LOG_TO_LOGGER` is generally used in free functions or member methods of the class that does not inherit `Loggable`. It is no different from ordinary ENVOY_LOG in the format.

Reply:
hello, what's the conclusion？

Reply:
Thanks for the explanation, shim